docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -119,6 +119,22 @@ If you are searching for a well tested solution, with a large community and nice
 
 Learn more about it on the dedicated [Nuxt](./nuxt.md) page.
 
+## Quasar Framework
+
+[Quasar](https://quasar.dev) with its CLI mode of operation, allows for you to build a SPA or PWA and have it served from the server initially. Quasar also allows for your code to be used for other platforms, including Electron and for mobile via Cordova. 
```

Comment:
I think we can emphasize on the server rendering part a bit. Say it builds isomorphic apps, like how Nuxt does? It is very much squished between CLI, SPA, PWA and Electron. You can miss it if you blink hehe.

Also, I dont think mentioning Electron or Mobile is needed here, though its good for those who stumble upon this page first. Any ideas?

Reply:
I'd like not to compare Quasar to Nuxt. They are two different beasts. 

Yeah, I was trying to get more than one sentence in on Quasar here.  You are right, I can expand on the SSR capabilities. 

Scott